client/components/domains/use-my-domain/utilities/transfer-domain-action.ts

```diff
@@ -80,7 +81,12 @@ export const transferDomainAction: AuthCodeValidationHandler =
 					await startInboundTransfer( selectedSite.ID, domain, authCode );
 					page( domainManagementTransferIn( selectedSite.slug, domain ) );
 				} catch ( error ) {
-					onDone( { message: transferDomainError.GENERIC_ERROR } );
+					const errorMessage =
+						transferDomainError.GENERIC_ERROR +
+						' ' +
+						__( 'Error message: ' ) +
+						`"${ error.message }"`;
```

Comment:
Do we need to strip new lines from this message, for example, or take any precautions to format it properly without letting it break the page's flow?

Reply:
That's a good point! Though I think there won't be a problem because all the error messages in the last 90 days didn't have line breaks. 5ef7863e1a9d46d4e22bb4439877a8d5-logstash